examples/scene/main.go

```diff
@@ -121,6 +121,8 @@ func main() {
 			Position: pctk.NewPos(60, 20),
 			Color:    pctk.Magenta,
 		}).Wait()
+		// lets player move guybrush freely ;-)
+		app.Do(pctk.ActorSelectEgo{ActorName: "guybrush"})
```

Comment:
I would also add some flag in the app to indicate whether the control panel is active or not, and a command to enable and disable it. During a cinematic, we can disable the control panel, so it will be rendered as a full black background. And then enable it on this point in order to show the panel with the verbs, the inventory, the action line, etc.

Reply:
Done!